home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -0,0 +1,438 @@
+openapi: 3.0.0
+info:
+  title: Voting
+  description: This element contains voting units created by owner based on the OpenAPI 3.0 specification.
+  version: 1.0.0
+tags:
+  - name: voting
+    description: Votings management operations
+    externalDocs:
+      description: Find out more
+      url: http://swagger.io
+paths:
+  /api/votings :
```

Comment:
`api` prefix should be removed from the endpoint path. It doesn't describe the resource and should be common for all endpoints.